### PR TITLE
added profile data and builder function

### LINKED ISSUE #7: Footer and Profile Setup
~~Add a footer element and use the renderToDom function to render footer to all pages.~~

Add a profile element and use the renderToDom function to render profile to all pages.

### PR DESCRIPTION
**Pull request by austincparker**
added profile data and builder function

### CODE REVIEW

File: scripts/utils/render.js [hunk 1]
```diff
@@ -86,8 +88,46 @@ const footerBuilder = () => {
   `;
   
   renderToDom('#footerContainer', domString);
+}
+
+// const userObj = {
```
Comment (pmpurcell) [inline scripts/utils/render.js]: Remove comment lines 93 - 105

File: scripts/repo.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
-import { footerBuilder } from "./utils/render.js";
+import { footerBuilder, profileBuilder } from "./utils/render.js";
 
 const init = () => {
-    footerBuilder(),
+    footerBuilder()
+    profileBuilder()
     console.log("repositories");
```
Comment (TrinityChristiana) [inline scripts/repo.js]: Let's remove this console.log

**Review by TrinityChristiana — CHANGES_REQUESTED** [review]

**Review by TrinityChristiana — APPROVED** [review]
LGTM!

**Review by pmpurcell — APPROVED** [review]
Good job!

**Pull request merged**